src/installation/guides/fde.md

```diff
@@ -235,10 +243,10 @@ a new file at `/etc/dracut.conf.d/10-crypt.conf` with the following line:
 install_items+=" /boot/volume.key /etc/crypttab "
 ```
 
-Next, install the boot loader to the disk.
+Next, install the boot loader to the disk. If the EFI firmware is 32 bits: change target to =i386-efi
 
 ```
-# grub-install /dev/sda
+# grub-install --target=x86_64-efi --efi-directory=/boot/efi --bootloader-id=void_grub --boot-directory=/boot --recheck
```

Comment:
This has been added to the `chroot` guide. Could you instead link there? We want to avoid duplicated information. And this command is only for EFI systems, while this guide covers both EFI and BIOS setups.

Reply:
yes! the wiki suffered from too much duplication regarding "this is how to install via xbps-install into a chroot env"